designs/2022-community-eslint-org/README.md

```diff
@@ -0,0 +1,383 @@
+- Repo: No ESLint repos, only the ones mentioned below (owned by private
+  persons)
+- Start Date: 2022-05-31
+- RFC PR: <https://github.com/eslint/rfcs/pull/91>
+- Authors: [@aladdin-add](https://github.com/aladdin-add),
+  [@MichaelDeBoey](https://github.com/MichaelDeBoey) &
+  [@ota-meshi](https://github.com/ota-meshi)
+
+# `@eslint-community` GitHub organization
+
+## Summary
+
+<!-- One-paragraph explanation of the feature. -->
+
+Facilitate a GitHub organization (eg.
+[@eslint-community](https://github.com/eslint-community)) where community
+members can help ensure widely depended upon ESLint related packages stay up to
+date with newer ESLint releases and doesn't hold the wider community back.
+
+## Motivation
+
+<!-- Why are we doing this? What use cases does it support? What is the expected
+outcome? -->
+
+There are some widely depended upon ESLint related packages that aren't kept up
+to date with eg. new ESLint releases. The community would like a place to
+collectively step in and ensure that these doesn't hold the wider community
+back.
+
+## Detailed Design
+
+<!--
+   This is the bulk of the RFC.
+
+   Explain the design with enough detail that someone familiar with ESLint
+   can implement it by reading this document. Please get into specifics
+   of your approach, corner cases, and examples of how the change will be
+   used. Be sure to define any new terms in this section.
+-->
+
+### How do you envision `@eslint-community`?
+
+Just like the [@jest-community](https://github.com/jest-community) org, the
+`@eslint-community` org would be a centralized home for ESLint related packages
+that don't belong in the main org or that are widely depended upon, aren't
+maintained anymore and are given back to the community.
+
+> Community repos for ESLint related projects
+
+The benefit of having the org in place would be that the community isn't
+dependent on one person's GitHub/`npm` account.  
+The added benefit is that the people maintaining the packages in the new org
+_could_ create a structure to streamline the packages more. They _could_ for
+instance decide things like using the same tools, supporting the same
+Node/ESLint versions, if we want to release all "rescued" packages under the
+`@eslint-community` `npm` scope or not, ...
```

Comment:
It seems like there's ambiguity around when we should move an NPM package to the `@eslint-community` NPM scope.

On the one hand, it would be nice to keep the existing NPM package name whenever possible to avoid churn so the thousands or millions of users of the package don't have to manually switch from the old name to the new name (the majority of users might never even get around to doing this, resulting in substantial fragmentation). This is my preference.

On the other, there might be organizational benefits to moving packages to scopes?

Reply:
I don't think there's much benefit to moving an npm package name. I agree whenever possible we'd want to avoid churn, so the only use case there would be when the original name was somehow unrecoverable.

Reply:
Was about to write the same as @ljharb 👍 Only move when it needs to move

Reply:
We should aim to get the original repo/npm package, if that's not possible, we should go with a fork and/or publish it under the org namespace

> If a package is meeting these criteria, but we can't get into contact with the
> owner of the repo to transfer ownership over to the new org, we could
> temporarily fork the repo into the new org. However, the ultimate goal would
> always be to transfer the original repo to the new org.  
> If we had a temporary fork and are able to transfer the original repo to the new
> org, we should first create PRs to the original repo to bring it up to date with
> the fork before continuing with other tasks.